src/commands/lint.ts

```diff
@@ -17,117 +17,105 @@
 // unused code. Any imports that are only used as types will be removed from the
 // output JS and so not result in a require() statement.
 
-import * as commandLineArgs from 'command-line-args';
-import * as logging from 'plylog';
+import * as chalkTypeOnly from 'chalk';
+import {ArgDescriptor} from 'command-line-args';
+import {UsageGroup} from 'command-line-usage';
+import * as lintLibTypeOnly from 'polymer-linter';
 import {ProjectConfig} from 'polymer-project-config';
+import * as lintImplementationTypeOnly from '../lint/lint';
 
-import {Command, CommandOptions} from './command';
+import {Command} from './command';
 
-const logger = logging.getLogger('cli.lint');
+export interface Options {
+  rules?: string[];
+  input?: string[];
+}
 
 export class LintCommand implements Command {
-  name = 'lint';
+  // TODO(rictic): rename to 'lint' here and elsewhere, delete
+  // legacy-lint.ts. Also update the README.
+  name = 'experimental-lint';
 
-  description = 'Lints the project';
+  description = 'Identifies potential errors in your code.';
 
-  args = [
+  args: ArgDescriptor[] = [
     {
       name: 'input',
       type: String,
       alias: 'i',
       defaultOption: true,
       multiple: true,
-      description: 'Files and/or folders to lint. Exclusive. Defaults to cwd.'
-    },
-    {
-      name: 'policy',
-      type: String,
-      alias: 'p',
-      description: 'Your jsconf.json policy file.',
-      defaultValue: false
-    },
-    {
-      name: 'config-file',
-      type: String,
-      defaultValue: 'bower.json',
-      description: (
-          'If inputs are specified, look for `config-field` in this JSON file.')
+      description: 'Files to lint. If given, these files will be the only ' +
+          'ones linted, otherwise all files in the project will be linted.'
```

Comment:
does this lint ALL files in your project, or just the ones reachable in the app described by polymer.json? Consider rewording if it's the latter.